[Tracing] Document OpenShift tempo url provider (#916)
kiali/kiali.io: content/en/docs/Configuration/p8s-jaeger-grafana/tracing/tempo.md

```

> Kiali will use the _external_url_ to redirect to the Tracing UI, in the "View in tracing" links. 

Reviewer: ```suggestion
Kiali uses the _external_url_ to construct "View in tracing links in the UI. 
```

> For the Tempo provider, by default, _url_format_ is set to _jaeger_ (Changed from Grafana in Kiali 2.17).

Reviewer: ```suggestion
For the Tempo provider the default _url_format_ is _jaeger_ (changed from _grafana_ in Kiali v2.17).
```

> Kiali will use the _external_url_ configuration to create the redirection link. The URL will have the Jaeger UI format to redirect to specific services and traces.

Reviewer: ```suggestion
So, by default the URL will have the Jaeger UI format when linking to specific services and traces.
```

> When the url_format is set to _grafana_, Kiali will use the _external_url_ set in the Grafana section, such as this example:

Reviewer: ```suggestion
If `spec.external_services.tracing.tempo_config.url_format` is set to _grafana_, Kiali will use the _external_url_ set in the `spec.external_services.grafana` section, such as this example:
```

      external_url: https://grafana.apps-crc.testing/
```

> It is also possible to set _url_format_ to "openshift". In that case, the url will redirect to the UI Plugin in OpenShift. 

Reviewer: ```suggestion
It is also possible to set _url_format_ to _openshift_. In this case the URL will redirect to the UI Plugin in the OpenShift console.
```
